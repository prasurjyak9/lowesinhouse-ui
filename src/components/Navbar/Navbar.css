.outer-nav{

    display: flex;

    flex-direction: row;

    height: 8vh;

    width: 100%;

    background-color:hsl(203, 67%, 98%, 1);

    text-align: center;

    /* display: inline-flex; */

    position: relative;

    font-size: 1.7vw;



    /* font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif; */

}

.inhouse

{



  text-decoration: none;

    color:#004990;

    font-size: 100%;

    font-weight: 600;

    padding: 3vh 0vw 0vh 0vw ;

    position: relative;

    top:1%;

    left: 10px;

}



.link{

    text-decoration: none;

    color: aliceblue;

    font-size: 90%;

    font-weight: 400;

     margin-top: 1vh; 

    padding: 2.5vh 1vw 1vh 2vw ;

    /* display: flex; */

}



#logo {

    height: 8vh;

    width: 12vw;

    padding: 1vh 3vw 1vh 1.5vw ;

    float: left;

  

    /* display: flex; */

}