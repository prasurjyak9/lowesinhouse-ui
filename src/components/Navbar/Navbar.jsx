import {Link} from "react-router-dom";
import "./Navbar.css"
import React from 'react';
export default function Navbar(){

    return (

        <div className='outer-nav'data-testid="navbar-test">

            <Link to = "/"  className='LandingPage' >

            <img src= "https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/Lowes_Companies_Logo.svg/1200px-Lowes_Companies_Logo.svg.png" className="loweslogo" alt="Lowes" id="logo"></img> 

            </Link>

            <h1 className='inhouse'><b>In House</b></h1>

        </div>

    )

};
