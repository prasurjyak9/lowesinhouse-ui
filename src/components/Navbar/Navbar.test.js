
import React from 'react'
import {render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { MemoryRouter } from 'react-router-dom';
import Navbar from './Navbar';

describe('Snapshot testing Navbar', () => {
    it("test", () =>{
        render(
            <MemoryRouter>
                <Navbar></Navbar>
            </MemoryRouter>
            )

        expect(screen.getByTestId("navbar-test")).toMatchSnapshot();
        
    })
})

