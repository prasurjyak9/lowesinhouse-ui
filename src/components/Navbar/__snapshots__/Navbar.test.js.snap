// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing Navbar test 1`] = `
<div
  class="outer-nav"
  data-testid="navbar-test"
>
  <a
    class="LandingPage"
    href="/"
  >
    <img
      alt="Lowes"
      class="loweslogo"
      id="logo"
      src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/Lowes_Companies_Logo.svg/1200px-Lowes_Companies_Logo.svg.png"
    />
  </a>
  <h1
    class="inhouse"
  >
    <b>
      In House
    </b>
  </h1>
</div>
`;
